### PR TITLE
Migrate page elements

### PR DESCRIPTION
**Pull request by OmarIthawi**

### CODE REVIEW

File: openedx/core/djangoapps/appsembler/sites/management/commands/migrate_page_elements.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+from openedx.core.djangoapps.appsembler.sites.utils import to_new_page_elements
+
+
+class Command(NoArgsCommand):
+    help = 'Use the language aware page elements JSON structure for all sites.'
+
+    def handle(self, **options):
+        for site_configs in SiteConfiguration.objects.exclude(site__id=settings.SITE_ID):
```
Comment (melvinsoft) [inline openedx/core/djangoapps/appsembler/sites/management/commands/migrate_page_elements.py]: Nice catch!

**Comment by OmarIthawi** [comment]
@melvinsoft I have designed the code to be safe for a second run. I think we should run it, and keep it for a while, until we're sure that we don't need it. After that, it should be safe to delete!

**Comment by melvinsoft** [comment]
@OmarIthawi Thanks for the answer! Let's get this merged!

**Comment by melvinsoft** [comment]
I've tried the new code in my devstack, and I'm getting the following errors:

The first one is with the default values. To reproduce it:
(Make sure you switched to this branch already)
- Create a new Tahoe site.
- Go to "Site Settings" -> "Site Languages" and add a new Language, French for example, but making sure you don't change the default language, and remain English.
- Go to "Page Editor" and edit the "Index Page", try to switch the language to French, you'll get an error.

**Comment by grozdanowski** [comment]
@OmarIthawi @melvinsoft ugh this took me more than it should. The reason for the fail is that Omar changed the variable name from `site_deafault_language` to `LANGUAGE_CODE` but the frontend was still looking for the old var name. I made a fix for that and made a PR: https://github.com/appsembler/amc/pull/114

**Pull request merged**